interni/norme-di-progetto/components/processi-di-supporto/revisione.tex

```diff
@@ -0,0 +1,75 @@
+\documentclass[../../norme-di-progetto.tex]{subfiles}
+
+\begin{document}
+
+\subsubsection{Finalità}%
+\label{subs:revisione/finalita}
+
+GruppOne istanzia il processo di revisione per sostenere le revisioni di avanzamento previste nel corso del progetto didattico.
+
+\subsubsection{Descrizione}%
+\label{subs:revisione/descrizione}
+
+Il processo viene utilizzato per determinare quanto il progetto sia svolto in allineamento con i requisiti, con la pianificazione e con il contratto.
+Richiede che un revisore valuti le attività o i prodotti del gruppo revisionato.
+Consiste delle seguenti attività:
+
+\begin{itemize}
+  \item Implementazione del processo
+  \item Revisione
+\end{itemize}
+
+\subsubsection{Attività}%
+\label{subs:revisione/attivita}
+
+\paragraph{Implementazione del processo}%
+\label{par:revisione/implementazione_del_processo}
+
+Parte dell'implementazione del processo è di responsabilità del revisore, che per GruppOne è il committente, e di conseguenza non controllabile direttamente.
```

Comment:
Qui diciamo che è di responsabilità del revisore, poche righe sotto diciamo che il revisore non ha alcuna responsabilità diretta: cambierei questa frase con "...del processo è svolta dal revisore, ..."

Reply:
> poche righe sotto diciamo che il revisore non ha alcuna responsabilità diretta

In realtà c'è scritto che "non ha alcuna responsabilità diretta _nel progetto_ [...]". Nella revisione alcune responsabilità le ha per forza.